models/issue.go

```diff
@@ -15,6 +15,7 @@ type Issue struct {
 	ID               uuid.UUID     `json:"id" db:"id"`
 	CreatedAt        time.Time     `json:"created_at" db:"created_at"`
 	UpdatedAt        time.Time     `json:"updated_at" db:"updated_at"`
+	GithubUpdatedAt  time.time		 `json:"github_updated_at" db:"github_updated_at"`
```

Comment:
Could you fix the formatting here? You can use go fit to fix all the formatting issues

Reply:
Where should I set the default sorting for queries?

Reply:
Hm, I don't find it in the buffalo docs, maybe they haven't implemented this feature yet. You can update this query https://github.com/ossn/fixme_backend/blob/master/actions/issues.go#L41 to order the issues based on the GithubUpdateAt

Reply:
Hi @alexdor, I cant find the syntax for orderby query over [here](https://gobuffalo.io/en/docs/db/querying/) 
Can you please guide me on how to do it?

Reply:
Unfortunately the documentation isn't complete. Try adding `.Order("github_updated_at desc")` to the query

Reply:
@himankpathak did the above help?

Reply:
yes it worked